src/app/SharpRaven/Data/SentryStacktrace.cs

```diff
@@ -91,7 +84,7 @@ public override string ToString()
 
             StringBuilder sb = new StringBuilder();
 
-            foreach (var frame in Frames)
+            foreach (var frame in Frames.Reverse())
```

Comment:
I would probably just remove the test, since it's not important for this library that the resulting string is exactly the same as the output of an `Exception`. Or change the test to only make sure `ToString` returns a non-empty string (and doesn't crash).